app-productivity/taskwarrior/autobuild/templates

```diff
@@ -0,0 +1,33 @@
+Template: taskwarrior/migrate_from_2
+Type: note
+Description: Upgrading from Taskwarrior 2 Needs Manual Migration
+ Taskwarrior 3 changed the way it stores and synchronises tasks, you would need
+ to export all your previous tasks before upgrading using the following command:
+   
+ $ task export > all-tasks.json
```

Comment:
```
    task export > all-tasks.json
```

With indentation

Reply:
If you have not yet exported your data, please select "No." After you have finished exporting your data, return and select "Yes" to proceed.